astro/src/content/blog/understanding-the-authorization-code-grant.mdx

```diff
@@ -0,0 +1,127 @@
+---
+publish_date: 2024-09-10
+title: Understanding The Authorization Code Grant
+description: We'll dive into the benefits of Authorization Code Grant in simplifying your app's authentication flows.
+authors: Alex Patterson
+categories: Education
+image: /img/blogs/video-auth-code-grant/authorization-code-grant-simple.png
+tags: oauth, code grant
+excerpt_separator: "{/* more */}"
+---
+import Aside from '../../components/Aside.astro';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+In this video, we will guide you through the intricacies of the [Authorization Code Grant flow](/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-sessions-refresh-tokens-cookies) using FusionAuth's sandbox environment. This simple example aims to be accessible to everyone, so follow along as we dive into the details.
+
+{/* more */}
+
+<YouTube id="mmyfwovnago" />
+
+## What Is The Authorization Code Grant?
+
+The Authorization Code Grant is a widely used method in OAuth 2.0 for enabling users to securely authenticate with third-party applications. For the tech nerds in the house that is [RFC6749](https://datatracker.ietf.org/doc/html/rfc6749). This method involves redirecting the user to an authorization server, obtaining an authorization code, and then exchanging that code for an access token. 
+Let's break this process down step-by-step.
+
+### Starting Point: The Initial URL
+
+Our journey begins by navigating to the admin application for FusionAuth. Initially, the URL [https://sandbox.fusionauth.io/admin](https://sandbox.fusionauth.io/admin) points to FusionAuth's admin UI, which then delegates authentication to FusionAuth itself. This delegation redirects us to the hosted login page that is both themed and customizable.
```

Comment:
Do you want to point to other methods to use FusionAuth? Docker, etc.

Reply:
this is part of the demo in the video, don't want to confuse much here for the post.